src/tests/index.test.ts

```diff
@@ -55,16 +51,17 @@ describe('Server', () => {
     server.close();
   });
 
-  it('listens to defaults to port 3000', () => {
-    delete process.env.PORT;
-    try {
-      const server = require('../index').default;
-      server.close();
-    } catch (err) {
-      // The app is likely already running but we don't need the server anyways
-    }
+  // it('listens on port 3000 as default port', () => {
```

Comment:
Since we're not running it, can we remove it? (Same with the commented out warning above)

Reply:
Meant to put this on all the lines, but I guess I only chose the first one. Basically all the commented out stuff not being used.

Reply:
Since it's in a test file, I'd rather leave it in only because I'd like to get it working eventually and it'd prevent someone from trying to make their own down the road. If you really want to remove it, I can open an issue to track it there but I don't think it has enough priority to warrant an issue haha